src/tribler-gui/tribler_gui/tests/test_core_manager.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import MagicMock, patch
```

Comment:
**nitpick:** It seems the tests don't use magic methods of mock objects (such as `__iter__`, `__getitem__`, etc.), so the base `Mock` class should be enough.

Reply:
unittest uses `MagicMock` as a default class for mock objects: https://docs.python.org/3/library/unittest.mock.html#the-mock-class

> The patch() decorators makes it easy to temporarily replace classes in a particular module with a Mock object. By default patch() will create a MagicMock for you. You can specify an alternative class of Mock using the new_callable argument to patch().